### PR TITLE
Add support to handle copybook with multiple root records and ASCII, EBCDIC conversion 

### PR DESCRIPTION
**Pull request by MohamedSabthar**
## Purpose

Fixes: https://github.com/ballerina-platform/ballerina-library/issues/6895
Fixes: https://github.com/ballerina-platform/ballerina-library/issues/6892

## Checklist
- [x] Linked to an issue
- [x] Updated the changelog
- [x] Added tests
- [ ] Updated the spec
- [x] Checked native-image compatibility

### CODE REVIEW

File: ballerina/ebcdic_ascii_transformer.bal [hunk 1]
```diff
@@ -0,0 +1,561 @@
+// Copyright (c) 2024 WSO2 LLC. (http://www.wso2.com) All Rights Reserved.
```
Comment (ayeshLK) [inline ballerina/ebcdic_ascii_transformer.bal:1]:
```suggestion
// Copyright (c) 2024 WSO2 LLC. (http://www.wso2.com).
```

File: ballerina/ebcdic_ascii_transformer.bal [hunk 2]
```diff
@@ -0,0 +1,561 @@
...
+    {ebcdic: 253, ascii: 26},
+    {ebcdic: 254, ascii: 26},
+    {ebcdic: 255, ascii: 26}
+];
+
+# Converts a byte array encoded in EBCDIC to a byte array encoded in ASCII.
+#
+# + ebcdic - The input byte array encoded in EBCDIC
+# + return - The output byte array encoded in ASCII
+public isolated function toAsciiBytes(byte[] ebcdic) returns byte[] {
```
Comment (ayeshLK) [inline ballerina/ebcdic_ascii_transformer.bal:551]: What if user provides an invalid encoded byte array ? Lets say user provides `ascii` encoded byte array here, what is the behavior ?

Comment (MohamedSabthar) [reply ballerina/ebcdic_ascii_transformer.bal:551]: it's users responsibility to pass the the correct encoded byte array. We don't have any mechanism to figure out the encoding as it's just bytes.

File: ballerina/ebcdic_ascii_transformer.bal (hunk 1 above)
Comment (MohamedSabthar) [reply ballerina/ebcdic_ascii_transformer.bal]: Fixed with [53c41d1](https://github.com/ballerina-platform/module-ballerinax-copybook/pull/40/commits/53c41d12d3e830803a466a480d96f56b86c987ed)

**Review by DimuthuMadushan — APPROVED** [review]

**Pull request merged**